clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -2152,6 +2153,8 @@ public void testSubscribeCompletedPartitionUnsubscribe(AAConfig aaConfig) throws
       doAnswer(invocation -> false).when(aggKafkaConsumerService).hasAnyConsumerAssignedForVersionTopic(any());
     }).setHybridStoreConfig(this.hybridStoreConfig).setExtraServerProperties(extraServerProperties);
     runTest(config);
+    HostLevelIngestionStats stats = storeIngestionTaskUnderTest.hostLevelIngestionStats;
+    verify(stats, timeout(TEST_TIMEOUT_MS)).recordStorageQuotaUsed(anyDouble());
```

Comment:
Can we have a check that consumer doesn't have any subscription first?
Then confirm that StorageQuotaUsed metric is still recorded later. For example, is it possible to check how many times this function has been called, like N times; then run an nondeterministic check to confirm the call times increase (>N).